blackduck/HubRestApi.py

```diff
@@ -86,11 +88,14 @@ class HubInstance(object):
 
     # TODO: What to do about the config file for thread-safety, concurrency
     configfile = ".restconfig.json"
-
+    global root_dir
+    # For refresh to find the .restconfig.json file upon reinitialization,
+    # client script must be executed in the same directory
+    root_dir = os.getcwd()
     def __init__(self, *args, **kwargs):
         # Config needs to be an instance variable for thread-safety, concurrent use of HubInstance()
         self.config = {}
-
+        self.read_config()
```

Comment:
why skip the try/except block?

Reply:
I think that try/except block is checking if the .restconfig exists, if it doesn't check the args and write it.

Reply:
I added a keyword argument named refresh_token that the client script can pass on instantiation. Refreshing the token is now off by default.